azurerm/internal/services/sentinel/sentinel_alert_rule_ms_security_incident_resource.go

```diff
@@ -196,28 +196,35 @@ func resourceSentinelAlertRuleMsSecurityIncidentCreateUpdate(d *schema.ResourceD
 	if !d.IsNewResource() {
 		resp, err := client.Get(ctx, workspaceID.ResourceGroup, "Microsoft.OperationalInsights", workspaceID.WorkspaceName, name)
 		if err != nil {
-			return fmt.Errorf("retrieving Sentinel Alert Rule Ms Security Incident %q (Resource Group %q / Workspace: %q): %+v", name, workspaceID.ResourceGroup, workspaceID.WorkspaceName, err)
+			return fmt.Errorf("retrieving Sentinel Alert Rule Ms Security Incident %q (Workspace: %q): %+v", name, workspaceID, err)
```

Comment:
heads up in this case this'll be rendered as:

```
(Workspace: "Workspace "name" (Resource Group %q)")
```

however if we need to build up the ID below anyway - we could likely reduce this down to:

```suggestion
			return fmt.Errorf("retrieving Ms Security Incident %s: %+v", id, err)
```

(same through the other errors here)